GitExtUtils/GitUI/Theming/ColorHelper.cs

```diff
@@ -186,7 +186,7 @@ public static bool IsLightTheme()
         }
 
         public static Color GetSplitterColor() =>
-            KnownColor.ControlLight.MakeBackgroundDarkerBy(0.035);
+            KnownColor.Window.MakeBackgroundDarkerBy(0);
```

Comment:
Hmm, I would expect `KnownColor.Window` without `MakeBackgroundDarker` to produce the same result.

Reply:
For a review, I don't know if this is correct?

Reply:
We transform KnownColor.Window to Color. If using without this I guess it may not change color when theme changed.